cpp/ps0002/ps0002solution.cpp

```diff
@@ -4,138 +4,35 @@
 #include "easylearn.cpp"
 using namespace std;
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
 int main()
 {
     //عدد سكان عمارة
-    short buildingResidents = 40;
+    unsigned short buildingResidents = 40; // 0.75/0.75
 
     //عدد البدايل لدوا في الصيدلية
-    short medicinAlternatives = 4;
+    unsigned short medicinAlternatives = 4; // 0.75/0.75
 
     //أفراد تيم في شركة برمجة
-    short teamMembers = 5;
+    unsigned short teamMembers = 5; // 0.75/0.75
 
     //سعر واحدة من مخبوزات الفرن
-    float bakedPrice = 10.5;
+    float bakedPrice = 10.5; // 0.75/0.75
 
     //الحرف الأول من اسم شخص
-    char personFirstLetter = 'h';
+    char personFirstLetter = 'h'; // 0.75/0.75
 
     //عدد المواليد في دولة تعدادها كبير زي الصين
-    long populationNumber = 2912534765;
+    unsigned int populationNumber = 2912534765; // 0.75/0.75
 
     //ناتج اكثر دقة لقسمة رقم على 7
-    double result = 83.8571429;
+    double result = 83.8571429; // 0.75/0.75
 
     //فاتورة الكهربا لخط انتاج في مصر
-    double electricityBill = 332188737363636.64564546;
+    double electricityBill = 332188737363636.64; // 0.75/0.75
 
     //عمر أشخاص كبار و صغار ف السن بالثانية
-    long ageInSeconds = 420480000;
+    long ageInSeconds = 420480000; // 0.75/0.75

```

Comment:
total 6.75/6.75
- [x] all variables required in the document are declared
- [x] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables